ui/valintalaskenta-ui/src/main/webapp/app/app-routes.js

```diff
@@ -45,4 +49,17 @@ angular.module('valintalaskenta')
 
         otherwise({redirectTo:'/haku/'});
 
-});
\ No newline at end of file
+    })
+    .run(function($rootScope, $location, HakukohdeModel) {
+        $rootScope.$on("$routeChangeStart", function(event, next, current) {
+            if (next.$$route.controllerAs == "valintakoetulos") {
+                // the only case of conditional redirect so far
+                HakukohdeModel.refreshIfNeeded(next.params.hakukohdeOid).then(function() {
+                    if (!HakukohdeModel.hasViite) {
+                        // "valintakoekutsut" tab should not be visible, jump to "persutiedot" tab
```

Comment:
Surely `perustiedot` :)

```suggestion
                        // "valintakoekutsut" tab should not be visible, jump to "perustiedot" tab
```

Reply:
Kyllä :)